src/mbgl/util/work_request.hpp

```diff
@@ -10,9 +10,10 @@ namespace mbgl {
 class WorkRequest : public util::noncopyable {
 public:
     using Future = std::future<void>;
+    using JoinedFlag = std::shared_ptr<std::atomic<bool>>;
```

Comment:
Why is this a `std::atomic<bool>`? This variable is never accessed from any but the Map thread.